com.unity.render-pipelines.high-definition/Documentation~/Compositor-User-Guide.md

```diff
@@ -49,3 +49,7 @@ The Render Schedule is a re-orderable list of Composition Layers and Sub-layers.
 ## Composition Parameters
 This section shows every exposed property that is not an input Composition Layer (for example, a Float to control the brightness of the final composition or a Color to tint a Texture2D). In this section, the window allows you to edit each property value outside of the Composition Graph. It is good practice to expose properties from the graph to the Graphics Compositor window, instead of hard-coding their values. This helps you to share composition profiles between Projects because those you do not need to open the Composition Graph to edit any values.
 
+## Composition of screen-space UI
+The Graphics Compositor can be used to compose screen-space canvas (UI) elements. However, in this case the camera sub-layer that draws the UI:
+- should have a unique camera assigned to it. The camera should not be used in any other layer
```

Comment:
- Should have a unique camera assigned to it. You should also not use the camera in any other layer.

Reply:
Here the second sentence is an "explanation" of what a "unique" camera means, so the term "also" is a bit confusing to me...
What about this one?:
- Should have a unique camera assigned to it. You should not use this camera in any other layer.

Reply:
Oh yeah, I see that now. Your suggestion looks good!

Reply:
Ok, should be good now.